modules/assignment/src/main/java/com/intuit/wasabi/assignment/cache/impl/AssignmentMetadataCacheTimeServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Copyright 2016 Intuit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.intuit.wasabi.assignment.cache.impl;
+
+import com.intuit.wasabi.assignment.cache.AssignmentMetadataCacheTimeService;
+
+import java.util.Date;
+
+/**
+ * Separate time service class so that cache functionality can be unit tested.
+ *
+ */
+public class AssignmentMetadataCacheTimeServiceImpl implements AssignmentMetadataCacheTimeService {
+    @Override
+    public Date getCurrentTime() {
```

Comment:
Prefer not to add default methods in interface as at the spirit of this feature it is intended for existing code where interface is being changed and for some reason updating all the implementations is not feasible.